recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
So, @jschueller I see a couple options for you at this point.

* Drop the `-j2`.
* Add your own `if` as [described]( https://github.com/conda-forge/staged-recipes/pull/531/files/8d97b7ba1b38350dee5af0f89571002f3418715b#r62074182 ).
* Help us figure out how to set this behind the scenes so that `CPU_COUNT` is correct.